storage-bigtable/src/lib.rs

```diff
@@ -399,9 +404,34 @@ impl Default for LedgerStorageConfig {
     }
 }
 
+const METRICS_REPORT_INTERVAL_MS: u64 = 10_000;
```

Comment:
This number was picked somewhat arbitrarily - any thoughts on this or seem reasonable ?

Reply:
It seems like a fine place to start. We can make it more fine-grained later, if we need to.